packages/core/content-type-builder/admin/src/components/ContentTypeBuilderNav/ContentTypeBuilderNav.tsx

```diff
@@ -16,11 +17,47 @@ import { useIntl } from 'react-intl';
 import { NavLink } from 'react-router-dom';
 import { styled } from 'styled-components';
 
+
 import { getTrad } from '../../utils/getTrad';
 
 import { useContentTypeBuilderMenu } from './useContentTypeBuilderMenu';
 
+
+// Custom hook to handle scroll locking
+const usePreventScroll = (ref: React.RefObject<HTMLDivElement>) => {
```

Comment:
if you have declared the hook in multiple places to follow the DRY principle is better to create a file just with the hook and import it wherever you need it